### ISSUE TITLE
[staging] Blue lines on hover

### ISSUE THREAD

**Issue by georai** [mozilla/voice-web#1156]
Blue lines to remove: https://www.dropbox.com/s/txh2rethtpmzrex/v6ieiasc.png?raw=1

@m-branson @Gregoor

**Comment by m-branson** [comment]
Looks like this is another focused state possibly for accessibility purposes? @georai did they show up on hover or after you clicked? I'm unable to reproduce in Nightly.

**Comment by georai** [comment]
@m-branson yes after click
